packages/api/src/hooks/useCreateNewRealAccount.ts

```diff
@@ -0,0 +1,28 @@
+import { useMemo } from 'react';
+import useMutation from '../useMutation';
+import useInvalidateQuery from '../useInvalidateQuery';
+
+/** A custom hook that creates a new real trading account. */
+const useCreateNewRealAccount = () => {
+    const invalidate = useInvalidateQuery();
+    const { data, ...rest } = useMutation('new_account_real', {
+        onSuccess: () => {
+            invalidate('authorize');
+        },
+    });
+
+    // Add additional information to the new real trading account response.
+    const modified_data = useMemo(() => {
+        if (!data?.new_account_real) return undefined;
```

Comment:
```suggestion
        if (!data?.new_account_real) return;
```
nit: there's no need to return undefined explicitly here

Reply:
Updated.